kinto_http/exceptions.py

```diff
@@ -15,3 +15,10 @@ def __init__(self, message=None, exception=None):
 
 class BucketNotFound(KintoException):
     pass
+
+
+class BackoffException(KintoException):
+
+    def __init__(message, retry_after):
+        message = message + '\nBackoff time:' + retry_after
+        super(message)
```

Comment:
With Python 2 you might have to do `super(BackoffException, self).__init__(message, exception)`